scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -156,13 +158,18 @@
 {% endif %}
 {% endmacro %}
 
-{% macro genome_build(case) %}
+{% macro genome_build(case, case_groups) %}
 <div href="#" class="bg-dark list-group-item text-white">
   <div class="d-flex w-100 justify-content-start align-items-center">
       <span class="fa fa-wrench fa-fw mr-3"></span>
       <span class="menu-collapsed">Genome build {{ case.genome_build }}</span>
   </div>
 </div>
+<div href="#" class="bg-dark list-group-item text-white">
+  <div class="d-flex w-100 justify-content-start align-items-center">
+    {{ igv_button(case,None,case_groups) }}
```

Comment:
Yes, I liked it: that's why I gave it a line of its own rather than just adding it as a link or a tiny button on the side of the build, although it might look a little lonely on the side there. Hm, maybe it would look better stretched over the whole sidebar, if it doesn't mess with the variant layout. 😁